iothub/device/src/ConnectionInfo.cs

```diff
@@ -15,13 +15,15 @@ internal ConnectionInfo()
             Status = ConnectionStatus.Disconnected;
             ChangeReason = ConnectionStatusChangeReason.ClientClose;
             StatusLastChangedOnUtc = DateTimeOffset.UtcNow;
+            RecommendedAction = RecommendedAction.DefaultAction;
         }
 
-        internal ConnectionInfo(ConnectionStatus status, ConnectionStatusChangeReason changeReason, DateTimeOffset changedOnUtc)
```

Comment:
It's unnecessary to pass changedOnUtc as a parameter into the constructor here, instead we can simply set it to `DateTimeOffset.UtcNow`